src/diff.rs

```diff
@@ -931,6 +931,61 @@ impl<'diff> DoubleEndedIterator for Deltas<'diff> {
 }
 impl<'diff> ExactSizeIterator for Deltas<'diff> {}
 
+/// Line origin constants.
+#[derive(Copy, Clone, Debug)]
+pub enum DiffLineType {
+    /// These values will be sent to `git_diff_line_cb` along with the line
+    Context,
+    /// ditto
```

Comment:
It would be great to add more context instead of **ditto**. 😀

<img width="546" alt="image" src="https://user-images.githubusercontent.com/14314532/111066342-7f284200-84f9-11eb-8114-cc7fa32e4bba.png">

Reply:
the problem is, that the official libgit2 docs do not document these either. so better remove the `ditto`?

Reply:
changed